tests/lib/nested.sh

```diff
@@ -123,13 +123,41 @@ is_classic_nested_system(){
     return 1
 }
 
+is_focal_system(){
+    if lsb_release -a | grep focal; then
+        return 0
+    fi
+    return 1
+}
+
+is_core_20_nested_system(){
```

Comment:
I see this is similiar to is_core_18_nested_system helper below, but I'm slightly confused about why do we have a helper that just wraps `if [ "$SPREAD_SYSTEM" = ...` check (without any extra logic).

Reply:
I think it makes the tests more readable personally

Reply:
In this case any the logic for determine which is hte nested system depends on the host system, so we check if the host system is bionic then the nested is core18 for core tests or bionic cloud for classic. 
Does it make sense?

Reply:
Ah, right, got it, thanks.